benchmarking/runs/test_saved_run.py

```diff
@@ -75,3 +64,30 @@ def test_saved_run():
                 simulation_outputs, "NumTeamsMeetingRequirements"
             ),
         )
+
+
+def _get_student_provider_settings(is_mock: bool):
```

Comment:
We can set this up as a convention, but remember that as we go, you and the team will be creating additional files within the `/runs` directory. So it might be better to just have separate files for mock vs real data runs as opposed to a single run doing both.

Also consider that with real data, the set of students is always the same in each trial whereas the set of students changes slightly per-trial with mock data - we might not want to lump these implications into a boolean flag like `is_mock`

Reply:
With that said I like this approach in general so let's revisit the idea when we write our first few "real" files in the `/runs` directory

Reply:
Should we create different folders for real/mock data then?

Reply:
I believe there already are, right? /data/simulated_data vs /data/real_data